appinventor/blocklyeditor/src/versioning.js

```diff
@@ -2982,6 +2983,22 @@ Blockly.Versioning.AllUpgradeMaps =
 
   }, // End Spinner upgraders
 
+  "Spreadsheet": {
+    2: [
+      Blockly.Versioning.changeMethodName("Spreadsheet", "ReadCol", "ReadColumn"),
+      Blockly.Versioning.changeMethodName("Spreadsheet", "WriteCol", "WriteColumn"),
+      Blockly.Versioning.changeMethodName("Spreadsheet", "AddCol", "AddColumn"),
+      Blockly.Versioning.changeMethodName("Spreadsheet", "RemoveCol", "RemoveColumn"),
+      Blockly.Versioning.changeEventName("Spreadsheet", "GotColData", "GotColumnData"),
```

Comment:
We talked about changing the names of these events to better match the naming conventions that appear elsewhere in App Inventor, which in turn better matches industry standard. Do we want to go ahead and do that here? E.g. ColumnWritten (WroteColumn?), ColumnAdded, etc. This looks more in line with what I see in TinyWebDB and CloudDB.

Reply:
I'm not sure that we want to hold up the release for that.